components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -106,68 +106,67 @@ clean {
 
 // Generate the Metrics API
 ext.gleanGenerateMetricsAPI = {
-    namespace -> {
-        variant ->
-
-        def sourceOutputDir =  "$buildDir/telemetry/src/${variant.name}/kotlin"
-        def generateKotlinAPI = task("generateMetricsSourceFor${variant.name.capitalize()}", type: Exec) {
-            description = "Generate the Kotlin code for the Metrics API"
-
-            workingDir  rootDir
-            // Note that the command line is OS dependant: on linux/mac is Miniconda3/bin/python.
-            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine new File(GLEAN_MINICONDA_DIR, "python")
-            } else {
-                commandLine new File(GLEAN_MINICONDA_DIR, "bin/python")
-            }
+    variant ->
+
+    def sourceOutputDir =  "$buildDir/telemetry/src/${variant.name}/kotlin"
+    def fullNamespace = "${variant.applicationId}.GleanMetrics"
```

Comment:
Is this going to work with libraries .. they do not really define an `applicationId`?

Reply:
I was surprised by this, too, but it seems that they do (somehow).  For the glean library it is set to `org.mozilla.components.service.glean`.  In another follow-on PR (#1926) I added another library and it did the right thing.  I also added glean support to fretboard in an exploratory branch and it did the right thing.